tonic/src/transport/channel/tls.rs

```diff
@@ -34,14 +34,14 @@ impl ClientTlsConfig {
         }
     }
 
-    /// Sets the CA Certificate against which to verify the server's TLS certificate.
+    /// Adds the CA Certificate against which to verify the server's TLS certificate.
     pub fn ca_certificate(self, ca_certificate: Certificate) -> Self {
         let mut certs = self.certs;
         certs.push(ca_certificate);
         ClientTlsConfig { certs, ..self }
     }
 
-    /// Sets the multiple CA Certificates against which to verify the server's TLS certificate.
+    /// Adds the multiple CA Certificates against which to verify the server's TLS certificate.
```

Comment:
Nit: "Adds the " -> "Add "?

Reply:
I followed the existing style of the documents of this struct's APIs, such as `Creates ...` of `ClientTlsConfig::new()`.